torch/_dynamo/eval_frame.py

```diff
@@ -1408,7 +1408,6 @@ def patch():
         disabled_multi_tensor_opt_modules = {
             adamax,
             radam,  # data-dependent control flow
-            sgd,  # for now, until we can speed up compilation (this affects the benchmarks)
```

Comment:
Would be good to understand the impact in runtime for benchmarks with this line of change. To my knowledge, a few hours for all the benchmarks is acceptable.